packages/components/psammead-most-read/src/index.stories.jsx

```diff
@@ -18,6 +18,8 @@ const listIndexRange = {
 };
 
 const pageTypes = ['oneColumn', 'twoColumn', 'multiColumn'];
+const rankTypeSize = { default: 'foolscap', trafalgar: 'trafalgar' };
+const linkTypeSize = { default: 'greatprimer', pica: 'pica' };
```

Comment:
In the drop-down, this reads `default` and `trafalgar `. As `default` isn't a typography class, would it be more appropriate for it to read `foolscap` and `trafalgar`?